src/main/java/uk/gov/hmcts/reform/bulkscan/payment/processor/service/servicebus/PaymentMessageParser.java

```diff
@@ -0,0 +1,42 @@
+package uk.gov.hmcts.reform.bulkscan.payment.processor.service.servicebus;
+
+import com.fasterxml.jackson.databind.MapperFeature;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.datatype.jsr310.JavaTimeModule;
+import com.microsoft.azure.servicebus.MessageBody;
+import org.springframework.util.CollectionUtils;
+import uk.gov.hmcts.reform.bulkscan.payment.processor.service.servicebus.exceptions.InvalidMessageException;
+import uk.gov.hmcts.reform.bulkscan.payment.processor.service.servicebus.model.PaymentMessage;
+
+import java.io.IOException;
+import java.util.List;
+
+public class PaymentMessageParser {
+
+    private static final ObjectMapper objectMapper;
+
+    static {
+        objectMapper = new ObjectMapper();
+        objectMapper.registerModule(new JavaTimeModule());
+        objectMapper.enable(MapperFeature.ACCEPT_CASE_INSENSITIVE_ENUMS);
+    }
```

Comment:
I think all this static setup can be done with the help of self returning method calls present in `ObjectMapper`:

```suggestion
    private static final ObjectMapper MAPPER = new ObjectMapper()
        .registerModule(new JavaTimeModule())
        .enable(MapperFeature.ACCEPT_CASE_INSENSITIVE_ENUMS);
```

Reply:
ObjectMapper instance is provided by Spring, we do not need to instantiate it ourselves, it is just waste of memory